mews-operations/availabilityBlockProcessor.md

```diff
@@ -0,0 +1,176 @@
+# Channel Manager API Operations: Process availability blocks
```

Comment:
There is no need to create a new page, we already have a page for Availability Blocks on the Mews side, which contains `processAvailabilityBlockConfirmation`. We should add the new operation to the existing page, unless there is a good reason not to?